### PR TITLE
update to three.js r76

### PR DESCRIPTION
**Pull request by kkananen**

### CODE REVIEW

File: browser/vendor/three/EditorControls.js [hunk 1]
```diff
@@ -183,7 +177,7 @@ THREE.EditorControls = function ( object, domElement ) {
 
 		event.preventDefault();
 
-		if ( scope.enabled === false ) return;
+		// if ( scope.enabled === false ) return;
```
Comment (kschzt) [inline browser/vendor/three/EditorControls.js]: pls remove commented code

File: browser/vendor/three/OBJLoader.js [hunk 2]
```diff
@@ -40,315 +65,442 @@ THREE.OBJLoader.prototype = {
...
 
-				// ["f 1//1 2//2 3//3", " 1//1", "1", "1", " 2//2", "2", "2", " 3//3", "3", "3", undefined, undefined, undefined]
+				} else {
 
-				addFace(
-					result[ 2 ], result[ 5 ], result[ 8 ], result[ 11 ],
-					undefined, undefined, undefined, undefined,
-					result[ 3 ], result[ 6 ], result[ 9 ], result[ 12 ]
-				);
+					throw new Error( "Unexpected face line: '" + line  + "'" );
```
Comment (kschzt) [inline browser/vendor/three/OBJLoader.js]: do we want to throw or ignore?

File: browser/vendor/three/OBJLoader.js [hunk 3]
```diff
@@ -40,315 +65,442 @@ THREE.OBJLoader.prototype = {
...
+					state.uvs.push(
+						parseFloat( result[ 1 ] ),
+						parseFloat( result[ 2 ] )
+					);
 
-			} else if ( ( result = normal_pattern.exec( line ) ) !== null ) {
+				} else {
 
-				// ["vn 1.0 2.0 3.0", "1.0", "2.0", "3.0"]
+					throw new Error( "Unexpected vertex/normal/uv line: '" + line  + "'" );
```
Comment (kschzt) [inline browser/vendor/three/OBJLoader.js]: throw

File: browser/vendor/three/OBJLoader.js [hunk 4]
```diff
@@ -40,315 +65,442 @@ THREE.OBJLoader.prototype = {
...
+				var value = result[ 1 ].trim().toLowerCase();
+				state.object.material.smooth = ( value === '1' || value === 'on' );
 
 			} else {
 
-				console.error("Ignoring unexpected line: " + line)
+				// Handle null terminated files without exception
+				if ( line === '\0' ) continue;
+
+				throw new Error( "Unexpected line: '" + line  + "'" );
```
Comment (kschzt) [inline browser/vendor/three/OBJLoader.js]: throw

File: browser/vendor/three/VRControls.js [hunk 5]
```diff
@@ -48,6 +50,14 @@ THREE.VRControls = function ( object, onError ) {
 
 	this.scale = 1;
 
+	// If true will use "standing space" coordinate system where y=0 is the
+	// floor and x=0, z=0 is the center of the room.
+	this.standing = false;
```
Comment (kschzt) [inline browser/vendor/three/VRControls.js]: should we set this on?

**Comment by kkananen** [comment]
replaced by https://github.com/vizorvr/vizor/pull/1711

**Pull request closed without merging**